docs/tracing_reference.rst

```diff
@@ -0,0 +1,156 @@
+.. _aiohttp-tracing-reference:
+
+Tracing Reference
+=================
+
+.. module:: aiohttp
+.. currentmodule:: aiohttp
+
+
+TraceConfig
+-----------
+
+Trace config is the configuration object used to trace requests launched by
+a Client session object using different events related to different parts of
+the request flow.
+
+.. class:: TraceConfig(trace_context_class=SimpleNamespace)
+
+   :param trace_context_class: factory used to create trace contexts,
+      default class used :class:`SimpleNamespace`
+
+   .. method:: trace_context()
+
+      Return a new trace context.
+
+   .. attribute:: on_request_start
+
+      Property that gives access to the signals that will be executed when a
+      request starts, based on the :class:`~signals.Signal` implementation.
+
+      The coroutines listening will receive as a param the `session`,
```

Comment:
Spinx prefer double backticks, e.g. ``` `session` should be replaced by ``session`` ```.
On other hand CPython documentation uses `*session*` for parameters bu looks like in aiohttp docs we have not pronounced agreement to use double backticks.

Reply:
Fixed, using `` as you proposed.